DESCRIPTION

```diff
@@ -1,11 +1,11 @@
 Package: funkyheatmap
 Title: Generating Funky Heatmaps for Data Frames
-Description: Allows generating heatmap-like visualisations for benchmark data 
+Description: Allows generating heatmap-like visualisations for data 
   frames. Funky heatmaps can be fine-tuned by providing annotations of the 
   columns and rows, which allows assigning multiple palettes or geometries
   or grouping rows and columns together in categories.
   Saelens et al. (2019) <doi:10.1038/s41587-019-0071-9>.
-Version: 0.6.0
+Version: 0.5.0.9000
```

Comment:
I disagree, 
1. this should be a minor version bump, so 0.7.0 (adding backwards compatible functionality)
2. if we want to align versions with funkyheatmappy it should be 0.7.0 too
And if it's before 1.0.0, the semantic versioning guarantees nothing

Reply:
the 0.5.0.9000 is during devt -- will be bumped to 0.6.0 or 0.7.0 when the package is pushed to cran